shell/platform/android/platform_view_android_jni.cc

```diff
@@ -11,7 +11,19 @@
 #include "lib/ftl/arraysize.h"
 #include "lib/ftl/logging.h"
 
-#define PLATFORM_VIEW reinterpret_cast<PlatformViewAndroid*>(platform_view)
+template <typename T>
+class HeapSharedPtr {
```

Comment:
Can the `make_shared()` call and the wrapper be replaced by `new std::shared_ptr(new PlatformViewAndroid())`?